radix-engine/src/wasm/wasmi.rs

```diff
@@ -202,6 +226,27 @@ impl ScryptoWasmValidator for WasmiEngine {
     }
 }
 
+impl ScryptoWasmInstrumenter for WasmiEngine {
+    fn instrument(&mut self, code: &[u8]) -> Result<Vec<u8>, InstrumentError> {
+        let mut module =
+            parity_wasm::deserialize_buffer(code).expect("Unable to parse wasm module");
```

Comment:
Will need to add validation that code itself doesn't include calls to `TBD_FUNCTION_INDEX`. Maybe add as a TODO comment?

Reply:
Not sure if I agree. This function index is from the `env` module, separated from the Scrypto module. Are we concerned about Scryto burning fees for nothing?

Reply:
Yes, exactly. Or rather, it is an assumption we make I believe and if assumption is broken, it may potentially break things (e.g. if we may need to change fees down the line and recreate a new wasm file.